src/AdminPanel/modules/LoginPage/LoginFormFields.tsx

```diff
@@ -1,25 +1,18 @@
-import { ErrorMessage } from '@hookform/error-message';
-import { useFormContext } from 'react-hook-form';
-import { LoginFormFieldsNames, LoginFormTypes } from './types';
+import { Input } from '@UG/libs/components';
+import { LoginFormFieldsNames } from './types';
+import { styled } from '@mui/material/styles';
 
-export const LoginFormFields = () => {
-  const {
-    register,
-    formState: { errors },
-  } = useFormContext<LoginFormTypes>();
+const StyledInputsDiv = styled('div')`
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+`;
 
+export const LoginFormFields = () => {
   return (
-    <>
-      <label data-cy="username-label" htmlFor={LoginFormFieldsNames.USERNAME}>
-        Username:
-      </label>
-      <input data-cy="username-input" {...register(LoginFormFieldsNames.USERNAME)} />
-      <ErrorMessage data-cy="username-error" errors={errors} name={LoginFormFieldsNames.USERNAME} as="p" />
-      <label data-cy="password-label" htmlFor={LoginFormFieldsNames.PASSWORD}>
-        Password:
-      </label>
-      <input data-cy="password-input" type="password" {...register(LoginFormFieldsNames.PASSWORD)} />
-      <ErrorMessage data-cy="password-error" errors={errors} name={LoginFormFieldsNames.PASSWORD} as="p" />
-    </>
+    <StyledInputsDiv>
```

Comment:
actually I think that this could be renamed to something more like `StyledFormFields`

Reply:
Resolved